quantum/quantum_keycodes.h

```diff
@@ -90,494 +82,438 @@ enum quantum_keycodes {
+
+    // Combos
+    CMB_ON,   // 5CF8
+    CMB_OFF,  // 5CF9
+    CMB_TOG,  // 5CFA
+
+    // Magic (continued)
+    MAGIC_SWAP_LCTL_LGUI,    // 5CFB
+    MAGIC_SWAP_RCTL_RGUI,    // 5CFC
+    MAGIC_UNSWAP_LCTL_LGUI,  // 5CFD
+    MAGIC_UNSWAP_RCTL_RGUI,  // 5CFE
+    MAGIC_SWAP_CTL_GUI,      // 5CFF
+    MAGIC_UNSWAP_CTL_GUI,    // 5D00
+    MAGIC_TOGGLE_CTL_GUI,    // 5D01
+    MAGIC_EE_HANDS_LEFT,     // 5D02
+    MAGIC_EE_HANDS_RIGHT,    // 5D03
 
     // Dynamic Macros
-    DYN_REC_START1,
-    DYN_REC_START2,
-    DYN_REC_STOP,
-    DYN_MACRO_PLAY1,
-    DYN_MACRO_PLAY2,
-
-    JS_BUTTON0,
-    JS_BUTTON_MIN = JS_BUTTON0,
-    JS_BUTTON1,
-    JS_BUTTON2,
-    JS_BUTTON3,
-    JS_BUTTON4,
-    JS_BUTTON5,
-    JS_BUTTON6,
-    JS_BUTTON7,
-    JS_BUTTON8,
-    JS_BUTTON9,
-    JS_BUTTON10,
-    JS_BUTTON11,
-    JS_BUTTON12,
-    JS_BUTTON13,
-    JS_BUTTON14,
-    JS_BUTTON15,
-    JS_BUTTON16,
-    JS_BUTTON17,
-    JS_BUTTON18,
-    JS_BUTTON19,
-    JS_BUTTON20,
-    JS_BUTTON21,
-    JS_BUTTON22,
-    JS_BUTTON23,
-    JS_BUTTON24,
-    JS_BUTTON25,
-    JS_BUTTON26,
-    JS_BUTTON27,
-    JS_BUTTON28,
-    JS_BUTTON29,
-    JS_BUTTON30,
-    JS_BUTTON31,
-    JS_BUTTON_MAX = JS_BUTTON31,
-
-#if defined(SEQUENCER_ENABLE)
-    SQ_ON,
-    SQ_OFF,
-    SQ_TOG,
-
-    SQ_TMPD,  // Decrease tempo
-    SQ_TMPU,  // Increase tempo
+    DYN_REC_START1,   // 5D04
+    DYN_REC_START2,   // 5D05
+    DYN_REC_STOP,     // 5D06
+    DYN_MACRO_PLAY1,  // 5D07
+    DYN_MACRO_PLAY2,  // 5D08
+
+    // Joystick
+    JS_BUTTON0,   // 5D09
+    JS_BUTTON1,   // 5D0A
+    JS_BUTTON2,   // 5D0B
+    JS_BUTTON3,   // 5D0C
+    JS_BUTTON4,   // 5D0D
+    JS_BUTTON5,   // 5D0E
+    JS_BUTTON6,   // 5D0F
+    JS_BUTTON7,   // 5D10
+    JS_BUTTON8,   // 5D11
+    JS_BUTTON9,   // 5D12
+    JS_BUTTON10,  // 5D13
+    JS_BUTTON11,  // 5D14
+    JS_BUTTON12,  // 5D15
+    JS_BUTTON13,  // 5D16
+    JS_BUTTON14,  // 5D17
+    JS_BUTTON15,  // 5D18
+    JS_BUTTON16,  // 5D19
+    JS_BUTTON17,  // 5D1A
+    JS_BUTTON18,  // 5D1B
+    JS_BUTTON19,  // 5D1C
+    JS_BUTTON20,  // 5D1D
+    JS_BUTTON21,  // 5D1E
+    JS_BUTTON22,  // 5D1F
+    JS_BUTTON23,  // 5D20
+    JS_BUTTON24,  // 5D21
+    JS_BUTTON25,  // 5D22
+    JS_BUTTON26,  // 5D23
+    JS_BUTTON27,  // 5D24
+    JS_BUTTON28,  // 5D25
+    JS_BUTTON29,  // 5D26
+    JS_BUTTON30,  // 5D27
+    JS_BUTTON31,  // 5D28
+
+    // Leader Key
+    KC_LEAD,  // 5D29
+
+    // Bluetooth: output selection (continued)
+    OUT_BT,  // 5D2A
+
+    // Lock Key
+    KC_LOCK,  // 5D2B
+
+    // Terminal
+    TERM_ON,   // 5D2C
+    TERM_OFF,  // 5D2D
+
+    // Sequencer
+    SQ_ON,  // 5D2E
+    SQ_OFF,  // 5D2F
+    SQ_TOG,  // 5D30
+
+    SQ_TMPD,  // 5D31
+    SQ_TMPU,  // 5D32
+
+    SQ_RESD,  // 5D33
+    SQ_RESU,  // 5D34
+
+    SQ_SALL,  // 5D35
+    SQ_SCLR,  // 5D36
+
+    SEQUENCER_STEP_MIN,  // 5D37
+    SEQUENCER_STEP_MAX = SEQUENCER_STEP_MIN + SEQUENCER_STEPS,
 
     SEQUENCER_RESOLUTION_MIN,
```

Comment:
Couldn't we move the sequencer block to `0x5B00`, as that should be enough keycodes for it?

Reply:
Yeah, but I think I'd rather save that range for something that takes a keycode argument, since otherwise we'd have to explicitly set the first keycode to `0x5B00`, then the next keycode after the sequencer ones to `0x5D2E`; in this case `SAFE_RANGE` but would need to be changed when someone adds new keycodes.

Reply:
Update: actually, it wouldn't fit there...
![image](https://user-images.githubusercontent.com/4781841/111238464-44362380-864b-11eb-8fe6-2c2d8f6890dc.png)

Reply:
Well, the steps itself could fit into that range, but .... that's not the only thing using a dynamic range, like this.